platform/broadcom/sonic-platform-modules-ruijie/b6510-48vs8cq/scripts/b6510-48vs8cq_platform.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#platform init script for B6510-48VS8CQ
+
+install_python_api_package() {
+    device="/usr/share/sonic/device"
+    platform=$(/usr/local/bin/sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
+
+    rv=$(pip install $device/$platform/sonic_platform-1.0-py2-none-any.whl)
```

Comment:
As mentioned above, SONiC is currently transitioning away from Python 2 to Python 3. Until we have completed the transition, please build and install both a Python 2 sonic-platform package (as you're doing here) as well as a Python 3 sonic-platform package.